spdx/writers/tagvalue.py

```diff
@@ -343,19 +341,27 @@ def write_document(document, out, validate=True):
     write_creation_info(document.creation_info, out)
     write_separators(out)
 
-    # Writesorted reviews
-    for review in sorted(document.reviews):
-        write_review(review, out)
-        write_separators(out)
+    # Write sorted reviews
+    if document.reviews:
+        out.write("# Reviews\n\n")
```

Comment:
Now that `write_separators` exists, shouldn't we use it here? Or are headlines handled differently?

Reply:
We could use it there but in my opinion this would make the code unnecessarily confusing.

Reply:
All right then